admin/config-ui/fields/class-field-environment.php

```diff
@@ -13,12 +13,14 @@ class WPSEO_Config_Field_Environment extends WPSEO_Config_Field_Choice {
 	public function __construct() {
 		parent::__construct( 'environment_type' );
 
-		/* Translators: %1$s resolves to the home_url of the blog. */
-		$this->set_property( 'label', sprintf( __( 'Please specify the environment in which this site - %1$s - is running.', 'wordpress-seo' ), get_home_url() ) );
+		$this->set_property( 'label', __( 'Please specify if your site is under construction or already active.', 'wordpress-seo' ) );
 
-		$this->add_choice( 'production', __( 'Production (this is a live site with real traffic)', 'wordpress-seo' ) );
-		$this->add_choice( 'staging', __( 'Staging (this is a copy of a live site used for testing purposes only)', 'wordpress-seo' ) );
-		$this->add_choice( 'development', __( 'Development (this site is running locally for development purposes)', 'wordpress-seo' ) );
+		$this->set_property( 'description',  __( 'Choose under construction if you want to keep the site out of the index
+ of search engines. Don\'t forget to activate it once you\'re ready to
+ publish your site.', 'wordpress-seo' ) );
+
+		$this->add_choice( 'production', __( 'Option A: My site is live and ready to be indexed', 'wordpress-seo' ) );
+		$this->add_choice( 'staging', __( 'Option B: My site is under construction and should not be indexed', 'wordpress-seo' ) );
```

Comment:
We are removing development here, which is okay, but I think we have to check where that value is used.

Reply:
This value isn't used anywhere at all. It was saved for potential future use.